hepcrawl/extractors/nlm.py

```diff
@@ -146,10 +146,10 @@ def get_page_numbers(node):
 
         fpage = node.xpath(".//FirstPage/text()").extract_first()
         lpage = node.xpath(".//LastPage/text()").extract_first()
-        if fpage and lpage:
+        try:
```

Comment:
this is already done by the [literature builder](https://github.com/inspirehep/inspire-schemas/blob/6307871ae8ab3809020767efa72f3c48b8dfb7fd/inspire_schemas/builders/literature.py#L450-L456), so not needed here.